src/components/Searchbar/Autocomplete.js

```diff
@@ -267,8 +315,21 @@ export default function Autocomplete({
         {...autocomplete.getFormProps({ inputElement: inputRef.current })}
       >
         <input ref={inputRef} className="aa-Input" {...inputProps} />
+        {inputProps.value.trim() !== '' ? (
+          <div className="aa-ClearButton" onClick={clearInput}>
+            <X size={18} weight="fill" />
+          </div>
+        ) : null}
+        <div onClick={handlePanelDropdown}>
+          <CaretDown size={14} weight="fill" color="#27272E54" />
```

Comment:
Can we use a theme var or pass down currentColor for hover states on the icon color?